templates/crud/generic_update_view.html

```diff
@@ -18,8 +18,8 @@
                             class="btn-submit mt-6">
                         Save
                     </button>
-                    <a href="javascript:history.back()"
-                       class="btn-cancel">Cancel</a>
+                    <a href="javascript:window.history.back()"
```

Comment:
Does this fix the issue with going back to the edit view rather than the db page?

Reply:
This was an attempt to fix an issue where the page gets reloaded if there was a validation error. Once we hit the back button it will go to the same page - and then we need to hit the back button again to get to the previous page. I know how to fix this in python but not sure about javascript. Do you know where the code to the submit button is? because the submit button fixes this